plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/delete/PositionDeleteWriter.java

```diff
@@ -114,4 +109,29 @@ public void abort()
     {
         writer.rollback();
     }
+
+    private void writeDeletes(ImmutableLongBitmapDataProvider rowsToDelete)
```

Comment:
It's unfortunate that we are forced to go through the overhead of converting bitmap into a parquet file (and then back for reads).
This is much better in delta-lake where we directly serialize the bitmap into a binary file.

Reply:
There is a proposal for Iceberg to do the same thing.